content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -224,14 +224,14 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
-### Backup Related Custom Resources Might Disappear
+### Backup Related Custom Resources In The Cluster And Backup Data On The Remote Backup Server Might Be Deleted
 
-Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+Longhorn will attempt to clean up the backup-related custom resources (BackupVolume, BackupBackingImage, SystemBackup, and Backup) in the following scenarios:
```

Comment:
Same. Need to emphasize the backup data will be deleted after Longhorn cleaned up the backup-related custom resources

Reply:
Modified. Thanks.